jdbc-util/src/main/java/com/salesforce/datacloud/jdbc/exception/DataCloudJDBCException.java

```diff
@@ -20,14 +20,14 @@
 
 @Getter
 public class DataCloudJDBCException extends SQLException {
+    private String fullCustomerMessage;
+
+    private String primaryMessage;
+
     private String customerHint;
 
     private String customerDetail;
```

Comment:
please add comment on what those represent and when they should be used. Get inspiration from the corresponding protobuf files